addons/survey/models/survey_question.py

```diff
@@ -600,7 +600,8 @@ class SurveyQuestionAnswer(models.Model):
     scoring_type = fields.Selection(related='question_id.scoring_type')
     # answer related fields
     value = fields.Char('Suggested value', translate=True, required=True)
-    value_image = fields.Image('Image', max_width=256, max_height=256)
+    value_image = fields.Image('Image', max_width=1024, max_height=1024)
+    value_image_name = fields.Char('Image Filename')
```

Comment:
Indeed, there may be some confusion. The field basically store the filename of uploaded images. this is useful when editing a row: you can see the image filename instead of the binary value (for instance 'image1.png' instead of 'ZEFfdsk453325fjdsfjo345FDdkjfdsf...') which can be a way to know which image is linked to which answer.

The part non implemented consisted of showing the name even when not editing a row, but is dropped because not supported (or too hackish / costly =) )

Reply:
Oh is there some kind of magic with the `filename="value_image_name"` attribute in the field definition?

First time I'm seeing this actually.

Could we rename to "value_image_filename" then maybe?

Reply:
Yes I think that when such a field receives an file, it stores its name in the given filename field if any. On rendering, if no name exists in the db, it will display the default rendering of the image field (until now, this is the size of the file). Good idea, renaming makes sense :)